integration/k8/README.md

```diff
@@ -77,7 +77,7 @@ set the `Type` to `simple`:
 $ sudo systemctl edit geopm
 $ cat /etc/systemd/system/geopm.service.d/override.conf
 [Service]
-Type=simple
+Type=notify
```

Comment:
```suggestion
Type=notify
TimeoutSec=900
```

Reply:
Put it in the default systemd unit file instead